### PR TITLE
update(doc): flesh out and add authors

### PR DESCRIPTION
**Pull request by nikrb**
Initially I added slack id's, names and github profiles, but then thought you might not want that, so please add what detail you want about yourselves.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,27 @@ TODO CI
...
+
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags).
+
+## Authors
+
+* **kofi** -
```
Comment (unrealkaii) [inline README.md]: I would love us to use our github usernames, apart from that. great start 👍

**Review by unrealkaii — CHANGES_REQUESTED** [review]
I would prefer using my github username only. apart from that. Great to start !

File: README.md (hunk 1 above)
Comment (unrealkaii) [inline README.md]:
```suggestion
 [@unrealkaii](https://github.com/unrealkaii)
```

**Comment by nikrb** [comment]
@unrealkaii the review is preventing this being merged. I thought commit the suggestion would give us green lights, not sure why it hasn't. Can you approve this so I can merge it?

**Review by unrealkaii — APPROVED** [review]

**Review by Francis-Njoku — APPROVED** [review]

**Pull request merged**